frontend/src/scenes/feature-flags/FeatureFlagProjects.tsx

```diff
@@ -0,0 +1,115 @@
+                        </LemonButton>
+                    </div>
+                </div>
+            }
+        >
+            <div className="max-w-120">
+                <Form
+                    logic={featureFlagLogic}
+                    formKey="ffCopyModal"
+                    id="ff-copy-modal-form"
+                    enableFormOnSubmit
+                    className="space-y-4"
+                >
+                    <div className="flex gap-2">
+                        <div className="flex gap-2 flex-1 flex-col">
+                            <div className="font-semibold leading-6 h-6">Feature Flag key</div>
+                            <div className="border px-3 rounded h-10 text-center flex items-center justify-center max-w-200">
+                                <span className="font-semibold truncate">{featureFlag.key}</span>
+                            </div>
+                        </div>
+                        <div className="flex gap-2 flex-1 flex-col items-center max-w-10">
+                            <div className="h-6" />
+                            <IconArrowRight className="h-10" fontSize="30" />
+                        </div>
+                        <Field name="Project" label="Project" className="flex-1">
+                            <LemonSelect
+                                value={currentOrganization?.teams?.[0].id}
+                                options={
+                                    currentOrganization?.teams?.map((org) => ({ value: org.id, label: org.name })) || []
+                                }
+                                className="min-w-40"
+                            />
+                        </Field>
+                    </div>
+                </Form>
+                <LemonBanner type="warning" className="mt-4">
+                    By performing the copy, you may overwrite your existing Feature Flag configuration in
+                    project-1-prod.
+                </LemonBanner>
+            </div>
+        </LemonModal>
+    )
+}
+
+const columns: LemonTableColumns<Record<string, string>> = [
+    {
+        title: 'Project',
+        dataIndex: 'project',
+    },
+    {
+        title: 'Flag status',
+        dataIndex: 'flag_status',
+    },
+]
+
+export default function FeatureFlagProjects(): JSX.Element {
+    const { openProjectsModal } = useActions(featureFlagLogic)
+
+    return (
+        <div>
+            <LemonButton
+                type="primary"
+                data-attr="copy-feature-flag"
+                className="mb-4"
+                fullWidth={false}
+                onClick={() => openProjectsModal()}
+            >
+                Copy to another project
+            </LemonButton>
+            <LemonTable
+                dataSource={records}
+                columns={columns}
+                emptyState="This feature flag is not being used in any other project."
+            />
+            <CopyFeatureFlagModal />
```

Comment:
if this is on it's own page, a modal doesn't make too much sense to me, can we inline it instead?

Reply:
weak suggestion^^

Reply:
Right, that makes a lot of sense :)